app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js

```diff
@@ -126,11 +126,8 @@ define([
             return this;
         },
 
-        /**
-         * Load data from server for shipping step
-         */
         navigate: function () {
-            //load data from server for shipping step
+            this.isVisible(true);
```

Comment:
It will not work because navigate method has a context of shipping component instead of step context. Remove context binding in https://github.com/amenk/magento2/blob/3f2295fc131d879b853e521fee3b989f234196a3/app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js#L89